src/lib.rs

```diff
@@ -28,10 +28,12 @@ pub mod layer;
 
 pub mod types;
 
+#[cfg(feature = "python-bindings")]
 use pyo3::prelude::*;
 
 /// Python bindings for packet dissection and sculpting in Rust (scalpel)
 #[pymodule]
+#[cfg(feature = "python-bindings")]
```

Comment:
I am not entirely sure, but shouldn't this line be before the previous line? That is shouldn't it read as - 

```rust
#[cfg(feature = "python-bindings")]
#[pymodule]
```
May be the current one perhaps compiles too - but the feature switch as discussed above will actually make more semantically correct?

Reply:
yeah, good point. this sequence is more accurate.

fixed b8cbd1d8187da4a306a0888668efef538f11c49e